ddl/ddl_api.go

```diff
@@ -3542,3 +3586,19 @@ type lockTablesArg struct {
 	SessionInfo   model.SessionInfo
 	IsCleanup     bool
 }
+
+// When handle charset and collate of a column, we should take collates(may multiple) in option into
+// consideration rather than handling it separately. Then the following option processing logic should
+// ignores this cause we will delete them from option here
+func extractCollateFromOption(def *ast.ColumnDef) []string {
```

Comment:
the golang comment needs start with the function name, `// extractCollateFromOption ....`

Reply:
ok. got it.

Reply:
> the golang comment needs start with the function name, `// extractCollateFromOption ....`

ok. got it.